src/utils/selectors/index.ts

```diff
@@ -55,3 +55,8 @@ export const createEffectBody = `${createEffectExpression} > ArrowFunctionExpres
 export const effectsImplicitReturn = `${mapOperatorsExpression} > ArrowFunctionExpression > ArrayExpression, ${mapToOperatorsExpression} ArrayExpression`
 
 export const effectsReturn = `${mapOperatorsExpression} ReturnStatement`
+
+const selectorsNames = 'createSelector|createFeatureSelector' // How to handle createSelectorFactory???
+export const selectorsExpression = `CallExpression[callee.name=/${selectorsNames}/]`
+const memorizedSelectorType = 'MemoizedSelector|MemoizedSelectorWithProps'
+export const memorizedSelector = `VariableDeclarator[id.typeAnnotation.typeAnnotation.typeName.name=/^${memorizedSelectorType}$/] > ObjectExpression > Property`
```

Comment:
This is my first attempt at eslint Typescript rules so I am not sure how to get a variable based on type

Any help would be appreciated

Reply:
Hi, first of all thank you for your contribution! 
I'll take a look later this week for this (I'm using https://astexplorer.net/ for most of this, but it seems like it's currently a bit broken).

I think to make things simpler; we should only scan for usages of `createSelector` (and not for variables that have `: MemoizedSelector<any, any>` types).

I'm also not sure if we can write this rule with a single query, we could scan for `createSelector` instead and then go up in the tree to find the variable declarator.

Reply:
Oh that's an interesting tool! I would not be surprised it needs to be more complicated.

If I have any luck I'll report back as well :)